roadmap-site/README.md

```diff
@@ -0,0 +1,109 @@
+# Challenges
+
+## User Object
+
+{
+"user_id":<memberid>,
+"first_name":string,
+"last_name":string,
+"yoe": number,
+"company": string,,
+"designation": string,
+"img": string,
+"github_id": string,
+"linkedin_id": string,
+"twitter_id": string,
+"instagram_id": string,
+}
+
+## **Requests**
+
+|               Route                |           Description           |
+| :--------------------------------: | :-----------------------------: |
+|  [GET /challenges](#get-challenges)| Returns all challenges          |
+|[POST /challenges](#post-challenges)|       Creates new challenge     |
+|[POST /challenge/subscribe](#post-subscribe) | subscribes users to challenge|
+
+## **GET /challenges**
+
+> returns all the active and completed challenges
+
+## Response
+
+```
+[
+    {
+        "id":"<challenge_id>",
+        "title": "Sherlock and Anagrams",
+        "level": "Easy",
+        "start_date": "10/05/2020",
+        "end_date": "10/25/2020",
+        "participants": [
+            <user_object>,
+            <user_object>
+        ],
+        "is_active": 1,
+        "is_user_subscribed":1
+    },
+    {
+        "id":"<challenge_id>",
+        "title": "Sherlock and Anagrams",
+        "level": "Easy",
+        "start_date": "10/05/2020",
+        "end_date": "10/25/2020",
+        "participants": [
+            <user_object>,
+            <user_object>
+        ],
+        "is_active": 0,
+        "is_user_subscribed":0
+    }
+]
+
+```
+
+## **POST /challenges**
+
+## Body
+
+```
+{
+    "title":"Sherlock and Anagrams",
+    "level":"Easy",
+    "start_date":"10/05/2020",
+    "end_date":"10/22/2020"
+}
+```
+
+> body will be in JSON format
```

Comment:
Sure about keeping the line number 78? Do we need to explicitly mention it?

Reply:
Just letting it know that I will be the passing the data in the json format.
I can also pass the header content type as `multipart/form-data` , then it might cause issues for you guys to handle it, so was just trying to be precise.